src/Nav/PrimaryNav/Mobile/index.js

```diff
@@ -1,39 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
-// import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+
+import CloseIcon from '../../../../img/close.svg';
```

Comment:
Ouch, any way to make an alias for `img/`?

Reply:
I tinkered with it a little but had trouble. Probably could but it's just about if it's worth it to spend the time. Do you want me to create a ticket for it?